dpdd.tex

```diff
@@ -687,9 +711,9 @@ \subsubsection{\SSObject Table}
 
 \subsubsection{Precovery Measurements}
 
-When a new \DIASource is detected, it's useful to perform PSF photometry at the location of the new source on images taken prior to discovery. These are colloquially known as \emph{precovery measurements}\footnote{When Solar System objects are concerned, precovery has a slightly different meaning: predicting the positions of newly identified \SSObjects on previously acquired visits, and associating with them the \DIASources consistent with these predictions.}.\dmreq{0287}\dmreq{0286} Performing precovery in real time over all previously acquired visits is too I/O intensive to be feasible. We therefore plan the following:
+When a new \DIASource is detected, it's useful to perform \newtext{forced} PSF photometry at the location of the new source on images taken prior to discovery. These are colloquially known as \emph{precovery measurements}\footnote{When Solar System objects are concerned, precovery has a slightly different meaning: predicting the positions of newly identified \SSObjects on previously acquired visits, and associating with them the \DIASources consistent with these predictions.}.\dmreq{0287}\dmreq{0286} Performing precovery in real time over all previously acquired visits is too I/O intensive to be feasible. We therefore plan the following:
 \begin{enumerate}
-\item For all newly discovered objects, perform precovery PSF photometry on visits taken over the previous 30 days\reqparam{precoveryWindow}\footnote{We will be maintaining a cache of $30$ days of processed images to support this feature.}.
+\item For all newly discovered objects, perform precovery PSF photometry on visits taken over the previous 30 days\reqparam{precoveryWindow}\footnote{We will be maintaining a cache of $30$ days of processed images to support this feature.}.  \newtext{These measurements will be stored in the \DIAForcedSource table.}
```

Comment:
Can we please start new sentences on a line of their own?  It's a lot easier to see the diff if we do that. DPDD is a nightmare to diff at times since the line lengths are so long that GitHub doesn't show highlighted text with the changed words. At least for new sentences we should move away from this.